lib/obs_github_deployments/deployment.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
```

Comment:
I believe this file should be under `lib/obs_github_deployments/commands/`, just like `version.rb`, right?

Reply:
Yes, specially with `zeitwerk` that's essential https://github.com/fxn/zeitwerk#file-structure

Reply:
Done

Reply:
Ah, in fact it should go under the `CLI` namespace... I'll fix it

Reply:
Done

Reply:
After talking with the rest of the team, we think this class should go in `lib/obs_github_deployments`, not inside `cli` or `commands` because this is a model, not some CLI stuff. So I'll revert the last commit.